src/epcpm/cantoxlsx.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import (
     annotations,
-)  # See PEP 563, check to remove in future Python version higher than 3.7
+)
+import re  # See PEP 563, check to remove in future Python version higher than 3.7
```

Comment:
Comment ended up on the wrong line.

```suggestion
)  # See PEP 563, check to remove in future Python version higher than 3.7
import re
```

Reply:
[94e593d](https://github.com/epcpower/pm/pull/85/commits/94e593d3eb6d081cada1def67e4e4fc6be7f73ac)